requirements.pip

```diff
@@ -0,0 +1,25 @@
+--extra-index-url https://staticfiles.krxd.net/foss/pypi/
+
+krux-boto==1.0.3
+
```

Comment:
Nit: Please put following comment here. You can see the actual example [here](https://github.com/krux/python-krux-boto-s3/blob/master/requirements.pip)

```
# Transitive libraries
# This is needed so there are no version conflicts when
# one downstream library does NOT specify the version it wants,
# and another one does.
```